Update meshstack.how-to.get-started-with-meshstack.md (#1056)
meshcloud/meshcloud-docs: docs/meshstack.how-to.get-started-with-meshstack.md

title: How to get started with meshStack
---

> This a guide that accompanies you on your very first stepts setting up a Internal Developer Platform with meshStack.

Reviewer: ```suggestion
This guide accompanies you to get started setting up a Internal Developer Platform with meshStack.
```

> **Prerequisites:**

> > 1. Your new meshStack was setup and the link to the meshPanel was provided to you.

Reviewer: ```suggestion
> 1. Your new meshStack is set up and the link to the meshPanel was provided to you.
```

> > 2. You need to be able to log into your meshStack to follow this guide.

Reviewer: ```suggestion
> 2. You need to be able to log into your meshStack and have admin area access.
```

## Step 1: Access the Admin Area of Your meshStack

The Admin Area is the central place to setup, customize and operate your very own Internal Developer Platform via meshStack.
To return to this view at a later point in time, you can always press the "Admin Area" button in the top bar.

## Step 2: Create Your First Workspace

> Workspaces provide you the opportunity to onboard and structure your use cases. Those workspaces can act as consumers and providers of services within the marketplace. The marketplace is 

Reviewer: ```suggestion
Workspaces provide you the opportunity to onboard and structure your use cases. Those workspaces can act as consumers and/or providers of services within the marketplace. The marketplace is 
```

the central place in meshStack to offer services that others consume. 

Now lets get started by creating the first workspace in your meshStack. To open the wizard, click on "Create Workspace +" as shown in the picture below.

> ![Create meshWorkspace](assets/create-workspace.png)

Reviewer: There's still an image here. Should we get rid of that?

> Of course, this can also be achieved via API. Take a look at our [workspace API](/api/index.html#_meshworkspace) to learn more.

Reviewer: As cool as this is, this should not be a part of a "Getting started" guide. Let's remove it.

Author: agreed!

> ## Step 2: Create your First Service 

Reviewer: ```suggestion
## Step 3: Create your First Service 
```

> The most central aspect of a Internal Developer Platform is offering services to other teams in your organization to jump start their projects, secure their application or extent their environments with additional ressources.

Reviewer: ```suggestion
The most central aspect of an Internal Developer Platform is offering services to other teams in your organization to jumpstart their projects, secure their application or extend their environments with additional resources.
```

> In order to create, test and publish your first service in form of a workspace level building block please follow this guide.

Reviewer: ```suggestion
```
Let's remove this. It's repeated below kind of

> In order to start the step by step guide for you first building block access the workspace and click on Service Management or Platform Builder on the top right corner.
> To offer your first service with meshStack, follow [AWS S3 Quickstart Guide](./meshstack.building-aws-quickstart-guide.md).

Reviewer: ```suggestion
If you want an easy starting point, open up the Service Management Area in your new workspace and follow the [AWS S3 Quickstart Guide](./meshstack.building-aws-quickstart-guide.md) where you will learn how to leverage meshStack and Terraform to offer a fully automated S3 bucket provisioning service in the marketplace.
```
